src/Features/VisualBasic/Portable/SignatureHelp/InvocationExpressionSignatureHelpProvider.MemberGroup.vb

```diff
@@ -32,8 +30,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.SignatureHelp
                 memberGroup = memberGroup.Where(Function(m) m.IsStatic)
             End If
 
-            Dim accessibleMembers = memberGroup.Where(Function(m) m.IsAccessibleWithin(within, throughType:=throughType)).ToList()
-            If accessibleMembers.Count = 0 Then
+            Return memberGroup.Where(Function(m) m.IsAccessibleWithin(within, throughType)).ToImmutableArray()
```

Comment:
`.WhereAsArray`?

Reply:
Correct me if I'm wrong, but I don't think ```WhereAsArray```` can be invoked here since memberGroup isn't an immutable array.